davealex/quick-credit
# #165588810 Design Admin View Specific Loan Application Page

#### What does this PR do?
This PR adds an admin_view_specific_loan.html page to the Project
#### Description of Task to be completed?
On this page, an admin can view the details on a specific loan application
#### How should this be manually tested?
Navigate to the admin_view_specific_loan.html page and ensure that the loan info card maintains a properly formatted structure on mobile devices
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#165588810
#### Screenshots (if appropriate)
#### Questions:
